contracts/contracts/NoLossDao.sol

```diff
@@ -236,18 +244,24 @@ contract NoLossDao is Initializable {
     proposalOwner[proposalId] = msg.sender;
     benefactorsProposal[msg.sender] = proposalId;
     state[proposalId] = ProposalState.Active;
+    iterationJoined[msg.sender] = proposalIteration;
     return proposalId;
   }
 
-  function withdrawProposal() public userHasActiveProposal(msg.sender) {
+  function withdrawProposal()
```

Comment:
We should think of having a common 'internal' function between the `withdrawProposal` and `withdrawUser`.